app/index.html

```diff
@@ -13,15 +13,33 @@
     <meta name="description" content="" />
     <meta name="keywords" content="" />
 
-    <meta property="og:title" content="dGRANTS – xxx" />
-    <meta property="og:description" content="xxx" />
-    <meta property="og:image" content="dgrants_social_preview.png" />
-    <meta property="og:url" content="https://xxx" />
+    <meta
+      property="og:title"
+      content="dGRANTS – Gitcoin Decentralized Grants"
+    />
+    <meta
+      property="og:description"
+      content="The decentralized Grants product is Gitcoin’s first move towards decentralization. Our vision is that all of our main grant rounds will be completely community-curated and operated. This protocol is the first step to enabling the web3.0 community to manage how their public goods are funded"
+    />
+    <meta
+      property="og:image"
+      content="https://grants.gtcdao.net/dgrants_social_preview.png"
```

Comment:
Is hosting down right now? If I visit this link in my browser it says "ipfs resolve -r /ipfs/bafybeigjtwtrrig7cbeag527pcsnjsz7n4s62j5g4lohcwjynxaiv2vtry/dgrants_social_preview.png: no link named "dgrants_social_preview.png" under bafybeigjtwtrrig7cbeag527pcsnjsz7n4s62j5g4lohcwjynxaiv2vtry"

Reply:
that's cause this image isn't there yet :P I had to change the location of the file in this PR